.github/workflows/regression.yml

```diff
@@ -13,11 +13,54 @@ on:
         description: Browser (can be chrome or firefox)
         default: chrome
         required: true
-
+      test:
+        description: Tests to run
+        default: all
+        options:
```

Comment:
Is this something we should add to when creating new tests?

Reply:
If we want the ability to run that specific newly added spec file, then yes.